Erosion:
  kernel_size: 5
Dilation:
  kernel_size: 5
